web/src/components/Task.tsx

```diff
@@ -49,6 +49,7 @@ export const Task = (props: Props): ReactElement => {
               href={`/tasks/${props.task.urlSlug}`}
               className={`usa-link margin-right-105 ${props.task.required ? "text-bold" : ""}`}
               data-task={props.task.id}
+              data-testid={props.task.id}
             >
```

Comment:
dust - Curious about if it would be preferable to use the props.task.name below instead. No strong opinions on my end.

Reply:
Cypress was being really finicky in what it wanted to pick up and this was the most assured way I could guarantee it would select the correct link.